docs/changelog.md

```diff
@@ -2,6 +2,11 @@
 
 ## CARET
 
+### v0.4.8 <small>\_ May 1, 2023</small> {id = "0.4.8"}
+
+- Improved implementation of message flow viewing API ([CARET_analyze #179](https://github.com/tier4/CARET_doc/pull/179)), ([CARET_analyze #274](https://github.com/tier4/CARET_analyze/pull/274)), ([CARET_analyze #282](https://github.com/tier4/CARET_analyze/pull/282))
```

Comment:
Since a API is replaced, the following description would be appropriate.

```suggestion
- Removed deprecated interface such as `message_flow` ([CARET_analyze #179](https://github.com/tier4/CARET_doc/pull/179)), ([CARET_analyze #274](https://github.com/tier4/CARET_analyze/pull/274)), ([CARET_analyze #282](https://github.com/tier4/CARET_analyze/pull/282))
```